docs/peertube-runner.md

```diff
@@ -0,0 +1,127 @@
+# PeerTube Runner
+
+This document describes how to use [django-peertube-runner-connector](https://github.com/openfun/django-peertube-runner-connector) library in order to use a [PeerTube runner](https://hub.docker.com/r/fundocker/peertube-runner/tags)  on marsha. The goal is to replace the current AWS transcoding to use PeerTube runner hosted by our servers instead.
+
+## Marsha integration
+
+### SocketIO
+
+`django-peertube-runner-connector` is using the [socketIO](https://socket.io/) library, this marsha needs to support its websocket endpoint.
```

Comment:
I don't understanf if, as a user, I have to modifiy something before using marsha ?

Reply:
No it is already done, this chapter is ambiguous, I am rewrite it a little